### PR TITLE
[74] update documents

### PR DESCRIPTION
**Pull request by jingkaimori**
## Why you open this Pull Request?

Update project 74 and corresponding commits.

## What work have you done in the current Pull Request?

- [x] Add task 74_2: large table crashes mogan
- [x] Mark task 74_1 as completed
- [x] Adjust layout of project 74
- [x] Update migration status of texmacs commits

### CODE REVIEW

File: devel/74_2.tm [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<TeXmacs|2.1.2>
+
+<style|<tuple|tmdoc|chinese|devel>>
```
Comment (darcy-shen) [inline devel/74_2.tm:3]: should be not chinese

Comment (darcy-shen) [reply devel/74_2.tm:3]: not a big problem

**Comment by darcy-shen** [comment]
layout engine具体是什么范围的代码？

**Comment by jingkaimori** [comment]
> layout engine具体是什么范围的代码？

[src/Typeset](https://github.com/XmacsLabs/mogan/tree/branch-1.2/src/Typeset) 和 [src/Style](https://github.com/XmacsLabs/mogan/tree/branch-1.2/src/Style) 文件夹下的代码

**Pull request merged**